### PR TITLE
Implement OnTimeoutPacket

### PR DESCRIPTION
**Pull request by anhductn2001**

### CODE REVIEW

File: x/feeabs/ibc_module.go [hunk 1]
```diff
@@ -201,7 +201,18 @@ func (am IBCModule) OnTimeoutPacket(
 	packet channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) error {
-	// TODO: Resend request if timeout
-	// TODO: emit event
+	chancap := am.keeper.GetCapability(ctx, host.ChannelCapabilityPath(am.keeper.GetPort(ctx), am.keeper.GetChannelId(ctx)))
```
Comment (GNaD13) [inline x/feeabs/ibc_module.go]: We not use this GetPort(ctx) and GetChannelId(ctx). ICQ port ID and Channel ID should in the params. You can reference to these [code](https://github.com/notional-labs/fee-abstraction/blob/3c6994e806a0e4abd50eccff46418c45615a39f9/x/feeabs/keeper/ibc.go#L98-L101). We should create a issue to remove these func

File: x/feeabs/keeper/keeper.go [hunk 2]
```diff
@@ -125,3 +127,13 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
...
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// OnTimeoutPacket resend packet when timeout
+func (k Keeper) OnTimeoutPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet exported.PacketI) error {
+	return k.channelKeeper.SendPacket(ctx, chanCap, packet)
+}
+
+func (k Keeper) GetCapability(ctx sdk.Context, name string) *capabilitytypes.Capability {
+	cap, _ := k.scopedKeeper.GetCapability(ctx, name)
```
Comment (GNaD13) [inline x/feeabs/keeper/keeper.go]: We should add the check here instead of _

File: x/feeabs/ibc_module.go [hunk 3]
```diff
@@ -201,7 +201,18 @@ func (am IBCModule) OnTimeoutPacket(
...
+	// Resend request if timeout
+	err := am.keeper.OnTimeoutPacket(ctx, chancap, packet) // If there is an error here we should still handle the timeout
+	if err != nil {
```
Comment (GNaD13) [inline x/feeabs/ibc_module.go:208]: We should add Logger here if func return an error. It will helpful for debug a lot

**Comment by anhductn2001** [comment]
@GNaD13 pls look it! tks

**Review by GNaD13 — APPROVED** [review]

**Comment by GNaD13** [comment]
LGTM, thank you. We still have some conflicts, can you resolve it?

**Pull request merged**